mmstudio/src/main/java/org/micromanager/internal/positionlist/TileCreatorDlg.java

```diff
@@ -364,6 +376,13 @@ public void actionPerformed(ActionEvent arg0) {
       studio.events().registerForEvents(this);
    }
 
+   @Subscribe
+   public void stuttingDown(ShutdownCommencingEvent se) {
```

Comment:
Is this needed, given that the profile is updated every time the values is edited?
Just checking, since if this is needed it might indicate an issue with the profile system.